### DISCUSSION TITLE
what are all the stamps

### DISCUSSION THREAD

**Discussion by diogo-soares** [community/community#19517] [Other Feature Feedback, Questions, & Ideas] [answered]
i would like to know what all the stamps i can get

**Comment by melvincwng** [comment, marked as answer]
@diogo-soares, are you referring to achievements?

If you are, please refer to [this link here](https://github.com/Schweinepriester/github-profile-achievements) for a list of achievements that can be obtained in GitHub.

Hope this clarifies.

If this answer has helped you, please kindly 'Mark as answer'.

**Reply by diogo-soares** [reply]
nice

**Reply by melvincwng** [reply]
@diogo-soares , Hi Diogo, thanks for marking this as answer. Appreciate it!

You can also **follow me** on GitHub too! Trying to expand my software developer connections on GitHub :)

Take care man, and have a nice day ahead 😄

**Comment by SamSambi** [comment]
Can I get a simple mail from you guys of the list I want to talk to you guys for 5 minutes please 
![Screenshot_20250619_005211](https://github.com/user-attachments/assets/a798b23c-f1db-4110-8ed4-83bc88d18e17)
Believe me or nothing I try all this just on this device I have 7/8 ideas like this can you guys help me with that list please help me to fulfill my dreams
